great_expectations/profile/basic_dataset_profiler.py

```diff
@@ -142,6 +135,8 @@ def _profile(cls, dataset):
                                                                        "value_ranges": [[None, None], [None, None], [None, None], [None, None], [None, None]]
                                                                    }
                                                                    )
+                    df.expect_column_kl_divergence_to_be_less_than(column, partition_object=None,
```

Comment:
Just for consistency sake, seems like this could still be SUMMARY?

Reply:
I think SUMMARY does not return the observed (?)